src/app-feature/account-feature/pages/login-page/login-page.component.ts

```diff
@@ -51,13 +52,14 @@ export class LoginPageComponent implements OnInit {
     }
 
     const credentials: IAccountCredentials = this.loginForm.getRawValue();
-    const onSuccess = (): void => {
-      const notificationConfig: INotificationConfiguration = {
-        message: 'TODO: Implement user authentication feature.',
-      };
-      this.context.ui.showNotification(notificationConfig);
-    };
+    this.accountService.login(credentials).subscribe(this.onLoginSuccess.bind(this));
+  }
 
-    this.accountService.login(credentials).subscribe(onSuccess);
+  private onLoginSuccess(): void {
+    this.context.state.initialize()
+    .pipe(take(1))
```

Comment:
No need to use `take(1)` when the data is provided by an API call, because API calls always complete, so there is one piece of data.